daemon/src/projection.rs

```diff
@@ -631,6 +631,10 @@ impl Tx {
 
         let _ = self.cfds.send(cfds_with_quote);
     }
+
+    fn update_quote(&self, quote: Option<bitmex_price_feed::Quote>) {
+        let _ = self.quote.send(quote.map(|q| q.into()));
```

Comment:
Should we care about the receiver potentially not being there anymore?  
I think it should not happen as both, the sender and receiver are both managed in `Actor`.

Reply:
I don't think we should care. I've only copied this code so didn't want to change any assumptions here!